tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py27,py27-pure,py33,py34,py35,pypy,pypy3,coverage
+    py27,py27-pure,py33,py34,py34-pure,py35,pypy,pypy3,coverage
```

Comment:
There is no specific reason why I have chosen this version. My goal was that at least one python3 version tests the python only implementation.
I think for completeness I would make sense to add a also a `py33-pure` and `py35-pure` possibly 3.6. I'll update the pull request in the next days.

Background:
Per default a test uses the C-Extension. In a `pure` environment the C-Extension is not compiled nor considered for the unittests. So only the `pure` environments test the python only implementation of ExtensionClass. Which means with the current envlist the python only implementation is tested in python27 and python34.